### ISSUE TITLE
Where can I download the boad dataset?

### ISSUE THREAD

**Issue by xiaoshuliu** [maiminh1996/YOLOv3-tensorflow#9]
Hi, I want to run the training code using the boat dataset first, it looks like the labels are in the git but the images are not. Where can I get them?

**Comment by geandwor** [comment]
Dea maiminh1996,

Could you please advise where I can download the boat data in the training? Thank you.

**Comment by Mr-DDDAlKilanny** [comment]
Any update?

**Comment by YunYang1994** [comment]
https://github.com/YunYang1994/tensorflow-yolov3 hope it helps you
